<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Nikita Sivakumar</title>
  <!-- import icons -->
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.0/css/all.min.css" integrity="sha512-yZcI9cXr..." crossorigin="anonymous" referrerpolicy="no-referrer" />
  <link rel="stylesheet" href="style.css" />
</head>
<body>
  <nav>
    <a href="#about">About</a>
    <!-- <a href="#interests">Interests</a> -->
    <a href="#skills">Skills</a>
    <a href="#projects">Projects</a>
    <a href="#teaching">Teaching</a>
    <!-- <a href="#talks">Talks</a>
    <a href="#awards">Awards</a> -->
  </nav>

  <section id="about">
    <div class="bio-container">
        <!-- LEFT COLUMN — photo + icons -->
        <div class="left-column">
            <img src="bio-photo.jpeg" alt="Nikita Sivakumar" class="about-pic">
            <div class="social-icons">
                <!-- LinkedIn -->
                <a href="https://www.linkedin.com/in/nikita-sivakumar-147a91190/" target="_blank" aria-label="LinkedIn">
                    <i class="fa-brands fa-square-linkedin"></i>
                </a>

                <!-- Google Scholar -->
                <a href="https://scholar.google.com/citations?user=v40a2iYAAAAJ&hl=en" target="_blank" aria-label="Google Scholar">
                    <i class="fa-brands fa-google-scholar"></i>
                </a>

                <!-- CV PDF -->
                <a href="CV_NikitaSivakumar.pdf" target="_blank" aria-label="CV">
                    <i class="fa-solid fa-file-pdf text-2xl"></i>
                </a>
            </div> 
        </div>
        <div class="about-text">
            <h2>Nikita Sivakumar</h2>
            <p>
                I am a PhD candidate in Biomedical Engineering at Johns Hopkins Univeristy. 
                Under the mentorship of Dr. Jude Phillip and Dr. Feilim Mac Gabhann, 
                I am developing data-driven and mechanistic computational pipelines to 1) identify single-cell
                lymphocyte motility behaviors and 2) map these behaviors to emergent function during the immune response.
                In addition to my thesis work,  I have had the opportunity to spearhead several projects that apply
                machine learning to improve precicision healthcare in collaboration with clinicians at Johns Hopkins.
                Prior to graduate school, I received a B.S. in Biomedical Engineering along with minors in computer science
                and engineering business from the University of Virginia (UVa). At UVa, I worked with Dr. Shayn
                Peirce-Cottler and Dr. Matthew Lazzara to create a multi-scale mechanistic model that elucidated
                how dynamic differential adhesion between cells induce emergent multi-cell patterning during development
                and disease. I am deeply passionate about integrating data-driven and mechanistic modeling approaches 
                to gain mechanistic insight into complex biological processes.
            </p>
        </div>
    </div>
  </section>

  <section id="skills">
    <h2>Skills</h2>
    <div class="skills-grid">
        <div class="skill">
            <img src="icons/abm.svg" class="skill-icon">
            <p>Agent-Based Modeling</p>
        </div>
        <div class="skill">
            <img src="icons/fx.svg" class="skill-icon">
            <p>ODE Modeling</p>
        </div>
        <div class="skill">
            <img src="icons/ml.png" class="skill-icon">
            <p>Machine Learning</p>
        </div>
        <div class="skill">
            <img src="icons/python.svg"  class="skill-icon">
            <p>Python</p>
        </div>
        <div class="skill">
            <img src="icons/java-logo.svg" class="skill-icon">
            <p>Java</p>
         </div>
        <div class="skill">
            <img src="icons/cplusplus.svg"  class="skill-icon">
            <p>C++</p>
         </div>
         <div class="skill">
            <img src="icons/cloud-data-internet.svg"  class="skill-icon">
            <p>HPC</p>
         </div>
        <div class="skill">
            <img src="icons/r.svg" class="skill-icon">
            <p>R</p>
         </div>
        <div class="skill">
            <img src="icons/mathworks.svg" class="skill-icon">
            <p>MatLab</p>
        </div>
    </div>
  </section>

  <section id="projects">
    <h2>Research Projects</h2>
    
    <div class="project-section">
        <!-- CENTERED TITLE -->
        <h3 class="project-title">
            Data-driven, mechanistic model elucidates how single-cell lymphocyte motility behaviors drive 
            B-cell : T-cell interactions during the antibdoy response.
        </h3>

        <div class="project-content">
            <div class="project-text">
                <p>
                    Lymphocytes rely on their ability to move to  surveil immune microenvironments and engage in critical
                    cell-cell interactions that drive the adaptive immune response. Within germinal centers,
                    B-cells and T-cells rely on motility to engage in  B-T interactions that drive antibody affinity maturation. 
                    To quantify how single-cell motility properties of lymphocytes enable their ability to interact with each other we developed PRISMM:
                    Pipeline for Recapitualating cell-cell Interactions from Spatial Motility Modelling:
                    <ul>
                        <li>PRISMM first uses unsupervised machine learning to identify emergent lymphocyte motility behaviors 
                        from three-dimensional, time-lapse intravital images of mouse germinal centers.</li>
                        <li>PRISMM then uses mechanistic, agent-based simulations to recapitulate observed motility behaviors 
                        at single-cell resolution and predict how these behaviors influence B-T interactions. </li>
                        <li>Our pipeline suggests that B-cells move slower, while T-cells take on faster and more exploratory trajectories. 
                        Moreover, our simulations indicate that slow-moving B-cells can experience comparable unique cell-cell interactions with T-cells 
                        as fast-moving B-cells. This suggests that the natural decision of B-cells to move slower may enable these cells 
                        to conserve energy for other processes without sacrificing surveillance ability.</li>
                    </ul>
                </p>
                <h3>Selected publications</h3>
                <ul>
                    <li>
                        <a href="project-cards/pdfs/Profiling_Dynamic_Patterns_of_Single_Cell_Motility.pdf" target="_blank">Maity, Sivakumar, Kamat, ..., Phllip. Profiling Dynamic Patterns of Single-Cell Motility. Advanced Sciences. (2024)</a>
                    </li>
                    <li>
                        <a href="project-cards/pdfs/biorxiv_PRISMM_manuscript.pdf" target="_blank">Sivakumar, Min, Choe, Beguelin, Phllip, Mac Gabhann. Data-driven simulations elucidate how lymphocyte motility behaviors drive cell-cell interactions with germinal centers. Biorxiv. (2025)</a>
                    </li>
                </ul>  
            </div>

            <div class="project-image">
                <img src="project-cards/images/phd-thesis.png" alt="Project 1 figure">
            </div>
        </div>
    </div>

    <div class="project-section">
        <!-- CENTERED TITLE -->
        <h3 class="project-title">
            Identifying emergent signatures of cell-free DNA methylation signatures.
        </h3>

        <div class="project-content">
            <div class="project-text">
                <p>
                    As our cells proliferate, DNA is naturally relesed by cells. Analyzing the methylation signatures of cell-free DNA 
                    collected by liquid biopsy can help us trace which tissues these cel-free DNA originated from.
                    Here we analyzed cell-free DNA methylation signatures from a cohort of RUSH patients. 
                    <ul>
                        <li>Unsupervised clustering of the signatures indicated three predominant signatures: erythrocyte-dominated, immune-dominated,
                            and cariovascular-dominated signatures.
                        </li>
                        <li>Patients with the cardiovascular cell-free DNA signature were more likely to develop cardiovascular disease.</li>
                        <li>Patients with cardiovascular and immune cell-free DNA signatures were more likely to develop frailty.</li>
                    </ul>
                </p>
                <h3>Selected publications</h3>
                <ul>
                    <li>
                        <a href="project-cards/pdfs/cell-free-DNA-frailty.pdf" target="_blank">Nidadavolu, Sosnowski, Sivakumar, ..., Abadir. The Journals of Gerentology. (2025)</a>
                    </li>
                </ul>
            </div>
            <div class="project-image">
                <img src="project-cards/images/frailty-ccf.png" alt="Project 1 figure">
            </div>
        </div>
    </div>

    <div class="project-section">
        <!-- CENTERED TITLE -->
        <h3 class="project-title">
            Machine learning to map standardly available clinical measurements to gold-standard pressure-volume loop phenotypes
            in pulmonary hypertension patients.
        </h3>

        <div class="project-content">
            <div class="project-text">
                <p>
                    Deriving pressure volume loops is the gold standard for measuring coupling between the right ventrial and pulmonary artery (RV - PA coupling)
                    in patients with pulmonary hypertension. While RV-PA coupling critically predicts prognosis, this is a highly invasive and expensive clinical procedure.
                    We sought to predict how standarly available measures of the heart (e.g. hemodynamics, MRI) could be predictive of RV-PA coupling states.
                    <ul>
                        <li>We applied unsupervised clustering to derive three novel PV-loop phenotypes with distinct times to clinical worsening</li>
                        <li>We trained a supervised machine learning model that integrated standardly available measurements of the heart to predict
                            PV-loop phenotypes with high accuracy.
                        </li>
                    </ul>
                </p>
                <h3>Selected publications</h3>
                <ul>
                    <li>
                        <a href="project-cards/pdfs/Unsupervised_Approach_to_Derive_Right_Ventricular_Pressure_Volume_Loop_Phenotypes.pdf" target="_blank">Sivakumar, Zhang, Chang-Chien, ..., Simpson. Pulmonary Circulation. (2025)</a>
                    </li>
                </ul>
            </div>
            <div class="project-image">
                <img src="project-cards/images/pcm.png" alt="Project 1 figure">
            </div>
        </div>
    </div>

    <div class="project-section">
        <!-- CENTERED TITLE -->
        <h3 class="project-title">
            Multi-scale modeling predicts how dynamically-induced differential adhesion between cells drives emergent multi-cell patterning.
        </h3>

        <div class="project-content">
            <div class="project-text">
                <p>
                    Dynamic cell-cell signaling processes induce differential adhesion between cells, which can lead to emergent tissue-level patterning during development and disease.
                    While we know that within a static system, differential adhesion behavior lead to more adhesive cell types being surrounded by less adhesive counterparts the role of 
                    cell-cell signaling in inducing these dynamic adhesion behaviors and its effect on multi-cell patterning was not well-characterized.
                    <ul>
                        <li>We developed an agent-based simulation of synthetic gene circuit in which initially non-adhesive cells engage in bi-directional signaling to induce 
                            the expression of distinct adhesion proteins. In our model, we can perturb cell-cell signaling parameters and differential adhesion parameters.</li>
                        <li>We validated that our simulations can accurately recapitulate experimentally observed multi-cell patterns.</li>
                        <li>We developed a novel sensitivity analysis pipeline that leverages unsupervised learning to identify emergent multi-cell patterns from our simulations. 
                            We then mapped cell-cell signaling and differential adhesion behaviors to the likelihood of forming distinct multi-cell patterns.
                        </li>
                    </ul>
                </p>
                <h3>Selected publications</h3>
                <ul>
                    <li>
                        <a href="project-cards/pdfs/pcb-multicell-spheroid-patterns.pdf" target="_blank">Sivakumar, Warner, Peirce, Lazzara. PLOS Computational Biology. (2022)</a>
                    </li>
                    <li>
                        <a href="project-cards/pdfs/fsysb_ml-abm.pdf" target="_blank">Sivakumar, Mura, Peirce. Frontiers in Systems Biology. (2022)</a>
                    </li>
                    <li>
                        <a href="project-cards/pdfs/multiscale-models-cancer.pdf" target="_blank">Warner, Sivakumar, Peirce, Lazzara. Current Opinion in Biomedical Engineering. (2019)</a>
                    </li>
                </ul>
            </div>
            <div class="project-image">
                <img src="project-cards/images/dif-adhesion-sim.png" alt="Project 1 figure">
            </div>
        </div>
    </div>

  </section>

  <section id="teaching">
    <h2>Teaching</h2>
    <p>I am deeply passionate about teaching! As an undergradaute and graduate student I have developed and taught 5 undergraduate courses 
        that focus on integrating data science and mechanistic modelling to answer inter-disciplinary ressearch questions. Recent courses include:</p>
        <ul>
            <li>Mechanistic & Data-Driven Insights into Disease. A two-week short-course focused on introducing students computational systems biology 
                approaches. Students learn and apply agent-based modeling, signaling network modeling, and deep learning to a series of biologial questions.
            </li>
            <li>Introduction to Data-Driven Computational Modeling. A two-credit, semester-long course focused on integrating machine learning and agent-based simulation
                to develop data-driven, mechanistic models. Students learn principles of ML, ABM, and their integration. They also embark in team projects where they develop
                their own computational model to answer a research question of their choice.
            </li>
        </ul>
  </section>

  <footer>
    <p>© 2025 Nikita Sivakumar</p>
  </footer>
</body>
</html>
